internal/api/inconsistencies_test.go

```diff
@@ -61,13 +68,15 @@ func TestInconsistencyReport(t *testing.T) {
 	}.Check(t, s.Handler)
 
 	// now put in some inconsistencies
-	s.MustDBExec(`UPDATE project_resources SET backend_quota = 10 WHERE project_id = $1 AND resource_id = $2`,
+	s.MustDBExec(`UPDATE project_az_resources SET backend_quota = 10 WHERE project_id = $1 AND az_resource_id IN (SELECT id FROM az_resources WHERE resource_id = $2 AND az = $3)`,
 		s.GetProjectID("dresden"),
 		s.GetResourceID("shared", "capacity"),
+		liquid.AvailabilityZoneTotal,
```

Comment:
```suggestion
	s.MustDBExec(`UPDATE project_az_resources SET backend_quota = 10 WHERE project_id = $1 AND az_resource_id = $2`,
		s.GetProjectID("dresden"),
		s.GetAZResourceID("shared", "capacity", liquid.AvailabilityZoneTotal),
```
Same directly below.